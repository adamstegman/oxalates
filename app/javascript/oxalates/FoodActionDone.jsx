import React from 'react';
import PropTypes from 'prop-types';

import "./FoodActionDone.scss";

export const FoodActionDone = ({ onDoneEditingFoods }) => {
  return (
    <button type="button"
            className="food-action"
            onClick={onDoneEditingFoods}>
      Done
    </button>
  );
};

FoodActionDone.propTypes = {
  onDoneEditingFoods: PropTypes.func.isRequired,
};
