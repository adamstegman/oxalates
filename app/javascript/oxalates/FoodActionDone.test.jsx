import React from 'react';
import renderer from 'react-test-renderer';

import { FoodActionDone } from './FoodActionDone';

test('FoodActionDone renders the done editing foods button', () => {
  const component = renderer.create(
    <FoodActionDone onDoneEditingFoods={() => {}} />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
