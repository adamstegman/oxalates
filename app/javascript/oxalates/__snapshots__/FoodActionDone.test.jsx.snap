// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FoodActionDone renders the done editing foods button 1`] = `
<button
  className="food-action"
  onClick={[Function]}
  type="button"
>
  Done
</button>
`;
